### PR TITLE
feat: make the help menu top 5 issues work and configurable

### PR DESCRIPTION
**Pull request by aronwk-aaron**
tested it, it just take html lol

image

### CODE REVIEW

File: dCommon/dConfig.cpp [hunk 1]
```diff
@@ -38,13 +38,10 @@ const std::string& dConfig::GetValue(std::string key) {
 }
 
 void dConfig::ProcessLine(const std::string& line) {
-	auto splitLine = GeneralUtils::SplitString(line, '=');
-
-	if (splitLine.size() != 2) return;
+	auto key = line.substr(0, line.find('='));
```
Comment (EmosewaMC) [inline dCommon/dConfig.cpp:41]: Could optimize this to just do one search

File: dNet/ClientPackets.cpp [hunk 2]
```diff
@@ -402,3 +403,33 @@ void ClientPackets::HandleChatModerationRequest(const SystemAddress& sysAddr, Pa
...
+	data.Insert("Summary4", Game::config->GetValue("help_4_summary"));
+
+	// Descriptions
+	data.Insert("Description0", Game::config->GetValue("help_0_description"));
+	data.Insert("Description1", Game::config->GetValue("help_1_description"));
+	data.Insert("Description2", Game::config->GetValue("help_2_description"));
+	data.Insert("Description3", Game::config->GetValue("help_3_description"));
+	data.Insert("Description4", Game::config->GetValue("help_4_description"));
+	
+	GameMessages::SendUIMessageServerToSingleClient(UserManager::Instance()->GetUser(sysAddr)->GetLastUsedChar()->GetEntity(), sysAddr, "UIHelpTop5", data);
```
Comment (EmosewaMC) [inline dNet/ClientPackets.cpp:433]: Check for null on most of these pointers

File: dWorldServer/WorldServer.cpp [hunk 3]
```diff
@@ -1297,6 +1297,12 @@ void HandlePacket(Packet* packet) {
 		break;
 	}
 
+
+	case eWorldMessageType::UI_HELP_TOP_5: {
+		ClientPackets::SendTop5HelpIssues(packet->systemAddress, packet);
```
Comment (EmosewaMC) [inline dWorldServer/WorldServer.cpp:1302]: Packet isn't used in the function, can remove it from the api

Comment (EmosewaMC) [reply dWorldServer/WorldServer.cpp:1302]: In that case just pass packet and read sysaddr when it is time

File: resources/worldconfig.ini
Comment (BrieFlory) [inline resources/worldconfig.ini:1]: If I don't fix spelling and grammar mistakes it'll literally kill me

File: resources/worldconfig.ini [hunk 4]
```diff
@@ -64,3 +64,15 @@ allow_nameplate_off=0
...
 # Turn logging of IP addresses for anti-cheat reporting on (1) or off(0)
 log_ip_addresses_for_anti_cheat=1
+
+help_0_summary=Got an issue?
+help_1_summary=Stuck loading?
+help_2_summary=Missing features?
+help_3_summary=Get smashed?
+help_4_summary=Want to contribute?
+
+help_0_description=Go to the DarkflameServer repository on Github to view issues and discussions about the server emulator!<br/><br/><a href="https://github.com/DarkflameUniverse/DarkflameServer"><font color="#0000EE">Click Here to go there!</font></a><br/><br/>
```
Comment (BrieFlory) [inline resources/worldconfig.ini:74]: GitHub

File: resources/worldconfig.ini [hunk 5]
```diff
@@ -64,3 +64,15 @@ allow_nameplate_off=0
...
+help_1_description=Try switching networks or using your phone's hotspot to resolve the issue<br/><br/>
+help_2_description=Yep, while the server emulator is mostly complete, there are still some unimplemted parts that are being worked on. Such as property behaviors and Dragonmaw Chasm to name a couple.<br/><br/>
+help_3_description=Skill issue!<br/><br/>
+help_4_description=Visit Discussions on DarkflameServer to<br>ask questions and collaborate with other devs!<br/><br/>
```
Comment (BrieFlory) [inline resources/worldconfig.ini:78]: Visit Discussions on the DarkflameServer GitHub page<br>to ask questions and collaborate with other devs!

File: resources/worldconfig.ini [hunk 6]
```diff
@@ -64,3 +64,15 @@ allow_nameplate_off=0
...
+
+help_0_summary=Got an issue?
+help_1_summary=Stuck loading?
+help_2_summary=Missing features?
+help_3_summary=Get smashed?
+help_4_summary=Want to contribute?
+
+help_0_description=Go to the DarkflameServer repository on Github to view issues and discussions about the server emulator!<br/><br/><a href="https://github.com/DarkflameUniverse/DarkflameServer"><font color="#0000EE">Click Here to go there!</font></a><br/><br/>
+help_1_description=Try switching networks or using your phone's hotspot to resolve the issue<br/><br/>
+help_2_description=Yep, while the server emulator is mostly complete, there are still some unimplemted parts that are being worked on. Such as property behaviors and Dragonmaw Chasm to name a couple.<br/><br/>
```
Comment (BrieFlory) [inline resources/worldconfig.ini:76]: unimplemented

Also, would change it to:

While DarkflameServer is a mostly complete emulator, there are still some features that aren't implemented. You can track these on the GitHub issues page.

File: resources/worldconfig.ini [hunk 7]
```diff
@@ -64,3 +64,15 @@ allow_nameplate_off=0
...
 log_ip_addresses_for_anti_cheat=1
+
+help_0_summary=Got an issue?
+help_1_summary=Stuck loading?
+help_2_summary=Missing features?
+help_3_summary=Get smashed?
+help_4_summary=Want to contribute?
+
+help_0_description=Go to the DarkflameServer repository on Github to view issues and discussions about the server emulator!<br/><br/><a href="https://github.com/DarkflameUniverse/DarkflameServer"><font color="#0000EE">Click Here to go there!</font></a><br/><br/>
+help_1_description=Try switching networks or using your phone's hotspot to resolve the issue<br/><br/>
```
Comment (BrieFlory) [inline resources/worldconfig.ini:75]: also suggest enabling a VPN?

**Review by BrieFlory — CHANGES_REQUESTED** [review]

File: resources/worldconfig.ini
Comment (aronwk-aaron) [reply resources/worldconfig.ini:1]: lol

File: resources/worldconfig.ini (hunk 7 above)
Comment (aronwk-aaron) [reply resources/worldconfig.ini:75]: true

File: resources/worldconfig.ini [hunk 8]
```diff
@@ -64,3 +64,15 @@ allow_nameplate_off=0
...
+help_1_summary=Stuck loading?
+help_2_summary=Missing features?
+help_3_summary=Get smashed?
+help_4_summary=Want to contribute?
+
+help_0_description=Go to the DarkflameServer repository on GitHub to view issues and discussions about the server emulator!<br/><br/><a href="https://github.com/DarkflameUniverse/DarkflameServer"><font color="#0000EE">Click Here to go there!</font></a><br/><br/>
+help_1_description=Try switching networks, using a VPN, or using your phone's hotspot to resolve the issue.<br/><br/>
+help_2_description=While DarkflameServer is a mostly complete emulator, there are still some features that aren't implemented. You can track these on the GitHub issues page.<br/><br/>
+help_3_description=Skill issue!<br/><br/>
+help_4_description=Visit Discussions on the DarkflameServer GitHub page<br/>to ask questions and collaborate with other devs!<br/><br/>
```
Comment (EmosewaMC) [inline resources/worldconfig.ini:78]: Newline

**Review by EmosewaMC — APPROVED** [review]

**Review by EmosewaMC — APPROVED** [review]

**Review by Jettford — APPROVED** [review]

**Pull request merged**